common.js

```diff
@@ -23,6 +23,18 @@ const encodeAmpersands = (str) => {
     .replace(/&gt;/g, "&amp;gt;");
 };
 
+const getDarkBg = () => {
+  const state = getState();
+  if (state.plugin_cfgs) {
+    let anyBsThemeCfg = state.plugin_cfgs["any-bootstrap-theme"];
+    if (!anyBsThemeCfg)
+      anyBsThemeCfg = state.plugin_cfgs["@saltcorn/any-bootstrap-theme"];
+    if (anyBsThemeCfg?.backgroundColorDark)
+      return anyBsThemeCfg.backgroundColorDark;
+  }
+  return "";
```

Comment:
tabler also has dark mode, and new themes in the future may as well. Can we add a default here? and ideally use some css variables as the fallback. you can try multiple css vars with `var(--var1, --var2)` and that can be nested. see https://github.com/saltcorn/saltcorn/blob/master/packages/server/public/saltcorn.css#L800 for an example

Reply:
Thanks. For Tabler, I'm now using --tblr-body-bg. any-bootstrap-theme has the body background as inline style, without its own css var, so I'm pulling it from the plugin cfg. The fallback is the tinymce dark skin, I just don't change it when const darkBg isn't set.

I'm not sure if we can use normal css vars everywhere. The editarea is in an iframe and the actual styles are defined in the parent, that's why I'm using the darkBg variable.